Bugzilla/Search.pm

```diff
@@ -2843,13 +2843,32 @@ sub _assignee_last_login {
 
 sub _component_nonchanged {
   my ($self, $args) = @_;
+  my $dbh = Bugzilla->dbh;
+  my (@products, @components);
+
+  # Allow to search product/component pairs like "Core::General, IPC"
+  foreach my $word (split(/[\s,]+/, $args->{value})) {
```

Comment:
I'm concerned that splitting on spaces here is not a good idea because many components include spaces. Some components consider commas too, but I'd be willing to deprecate that.
Alternatively, `Text::ParseWords` can be used.

Reply:
Ah true. Removing `\s` 😆